image-registry/remove-registry-baremetal.md

```diff
@@ -0,0 +1,137 @@
+---
+title: remove-registry-baremetal
+authors:
+  - "@adambkaplan"
+reviewers:
+  - "@dmage"
+  - "@bparees"
+  - "@sdodson"
+  - "@rgolangh" # RHV
+  - "@russellb" # k8s-native infra
+approvers:
+  - "@bparees"
+  - "@smarterclayton"
+  - "@derekwaynecarr"
+creation-date: 2019-09-25
+last-updated: 2019-09-25
+status: provisional
+see-also: 
+replaces:
+superseded-by:
+---
+
+# Remove Registry on Baremetal
+
+## Release Signoff Checklist
+
+- [ ] Enhancement is `implementable`
+- [ ] Design details are appropriately documented from clear requirements
+- [ ] Test plan is defined
+- [ ] Graduation criteria for dev preview, tech preview, GA
+- [ ] User-facing documentation is created in [openshift/docs]
+
+## Summary
+
+On `baremetal` platform clusters, the OpenShift image registry operator will bootstrap itself as `Removed`.
+This will allow `openshift-installer` to complete installations on this platform type.
+A separate process will be responsible for configuring storage and updating the image registry operator's state to `Managed`.
+
+## Motivation
+
+IPI installation options for Kubernetes-native infrastructure (formerly RHHI.next) and Red Hat Virtualization (RHV) have been declared a key initiative for OpenShift 4.3.
```

Comment:
I'm not terribly worried about the Image Registry being removed, however we need to be clear that an installation that requires post installation admin activities in order to configure the storage provider is *not* an IPI cluster.